examples/python/gymnasium_vect_bench.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python3
+
+#####################################################################
+# Test and benchmark the vectorization of VizDoom in gymnasium
+#####################################################################
+
+import time
+import argparse
+import warnings
+
+from gymnasium.vector import AsyncVectorEnv
+import gymnasium
+from vizdoom import gymnasium_wrapper
+
+warnings.filterwarnings("ignore")
+parser = argparse.ArgumentParser()
+parser.add_argument("--n_envs", type =int, default = 1, help ="Number of envs")
+args =parser.parse_args()
+seed = 42
+n_episodes = 1000
+
+# Pick an environment VizdoomCorridor-v0
+envs = gymnasium.make_vec("VizdoomCorridor-v0", num_envs =args.n_envs, vectorization_mode="async")
```

Comment:
It would be nice to have an argument for sync/async modes to make it easier to test them